cpp/modmesh/inout/inout_util.cpp

```diff
@@ -0,0 +1,36 @@
+#include <modmesh/inout/inout_util.hpp>
```

Comment:
Any license declaration in new files.  You need to use the same license term as the existing files.